src/extension/background-script/actions/ln/getBalance.js

```diff
@@ -0,0 +1,16 @@
+import state from "../../state";
+
+const getInfo = async (message, sender) => {
```

Comment:
what is this getBalance.js for? where is it used.
and I think you have a c&p error here => `getInfo`?